src/components/BaseModal.vue

```diff
@@ -2,33 +2,33 @@
   <Transition>
     <div
       v-if="isVisible"
-      class="base-modal-overlay"
+      class="base-modal__overlay"
```

Comment:
I'm not sure I understand when we are using __ in the class name vs. hyphen (-) is this a rule I should be following?